mindmeld/text_preparation/stemmers.py

```diff
@@ -74,46 +85,57 @@ def stem_word(self, word):
         return word if stem == "" else stem
 
 
-class NoOpStemmer(Stemmer):
-    @property
-    def _stemmer(self):
-        return
+class StemmerFactory:
+    """Stemmer Factory Class"""
 
-    def stem_word(self, word):
-        return word
+    @staticmethod
+    def get_stemmer(stemmer):
+        """A static method to get a stemmer.
 
+        Args:
+            translator (str): Name of the desired translator class
+        Returns:
+            (Translator): Translator Class
+        """
+        if stemmer == EnglishNLTKStemmer.__name__:
+            return EnglishNLTKStemmer()
+        if stemmer == SnowballNLTKStemmer.__name__:
+            return SnowballNLTKStemmer()
+        raise AssertionError(f" {stemmer} is not a valid 'stemmer'.")
```

Comment:
This is probably better conveyed as a `TypeError`.

Reply:
Makes sense, fixed